build-tools-internal/src/main/groovy/elasticsearch.build-scan.gradle

```diff
@@ -146,7 +144,7 @@ buildScan {
             'buildkite-agent',
             'annotate',
             '--context',
-            'gradle-build-scans',
+            result.failure ? 'gradle-build-scans-failed' : 'gradle-build-scans',
```

Comment:
These won't be collapsible unless you add this context here, but that may be okay since it's the failed ones: https://github.com/elastic/elasticsearch/blob/main/.buildkite/hooks/pre-command#L96

The other thing that could happen if you don't preemptively add the context is that the million gobld error annotations that come through sometimes could effectively push this error annotation out of view

Reply:
I think I prefer failures being expanded by default. Let's see how it plays out.

> The other thing that could happen if you don't preemptively add the context is that the million gobld error annotations that come through sometimes could effectively push this error annotation out of view

We need to escalate this with the CI team if we haven't already. It makes no sense to annotate our jobs with transient errors that aren't actionable by us.